draft-ietf-masque-connect-udp-listen.md

```diff
@@ -153,11 +163,114 @@ UDP Payload:
 : The unmodified UDP Payload of this proxied UDP packet (referred to as "data
 octets" in {{UDP}}).
 
+# Context ID {#contextid}
+
+A Context ID of value 0 represents no compression. When using context ID 0, the
```

Comment:
The no-compression context isn't 0 though right? It's the value allocated on line 111. Maybe we need to give that value a name up there and reuse that name here.

That said, another option could be for us to change the meaning of context ID 0 and use 0 instead of allocating one. But then we have some slight ambiguity when the client sends before receiving the server's HTTP headers (and knowing whether the server supports this extension or not), so we probably want to stick with a non-zero allocated value.